### PR TITLE
go1.7 vet fixes

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco

### CODE REVIEW

File: go/logger/standard.go [hunk 1]
```diff
@@ -337,10 +337,15 @@ func PickFirstError(errors ...error) error {
 }
 
 func (log *Standard) CloneWithAddedDepth(depth int) Logger {
-	clone := *log
+	clone := Standard{
+		filename:        log.filename,
+		module:          log.module,
+		externalHandler: log.externalHandler,
+	}
```
Comment (maxtaco) [inline go/logger/standard.go]: what's this one?

Comment (patrickxb) [inline go/logger/standard.go]: i don't know what it's for...i just copied the struct without copying the mutex.

Comment (maxtaco) [inline go/logger/standard.go]: Ah, ok.  I wonder if that breaks anything though....

Comment (patrickxb) [inline go/logger/standard.go]: Looks like KBFS uses it extensively.

@strib does this look ok?

Comment (strib) [inline go/logger/standard.go]: Looks fine to me.

**Comment by maxtaco** [comment]
👍

**Pull request merged**